### PR TITLE
Operation methods added, console calculator implemented, Unit tests

### PR DESCRIPTION
**Pull request by AlexeyScherbina**

### CODE REVIEW

File: NUnitTestProject1/OperationTest.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+{
+    public class OperationTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void TestSumm()
```
Comment (IvanKrt) [inline NUnitTestProject1/OperationTest.cs]: Incorrect naming.  Should be NameMethod_Action_Result

File: NUnitTestProject1/OperationTest.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        }
+        [Test]
+        public void TestNotEqualDivision()
+        {
+            double a = 1, b = 11;
+            double expected = 121;
+            double actual = Operations.Division(a, b);
+            Assert.AreNotEqual(expected, actual);
+        }
+
```
Comment (IvanKrt) [inline NUnitTestProject1/OperationTest.cs]: Empty spaces

File: SimpleCalc/Operations.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        public static double Subtraction(double a, double b)
+        {
+            return a - b;
+        }
+        public static double Multiplication(double a, double b)
+        {
+            return a * b;
+        }
+        public static double Division(double a, double b)
+        {
```
Comment (IvanKrt) [inline SimpleCalc/Operations.cs:24]: What if b is null?

**Review by IvanKrt — CHANGES_REQUESTED** [review]

File: NUnitTestProject1/OperationTest.cs [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+namespace Tests
+{
+    public class OperationTest
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+        [Test]
+        public void Summ_Numbers_Equal()
```
Comment (IvanKrt) [inline NUnitTestProject1/OperationTest.cs]: Should be Summ_TwoPositiveNumbers_ResultShouldBeEqual

File: NUnitTestProject1/OperationTest.cs [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+        {
+            double a = 5, b = 10;
+            double expected = 15;
+            double actual = Operations.Summ(a, b);
+            Assert.AreEqual(expected, actual);
```
Comment (IvanKrt) [inline NUnitTestProject1/OperationTest.cs]: Use Extensions. Should be expected.Should.Be(actual). Ask Artem

File: NUnitTestProject1/OperationTest.cs [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+        public void Division_Numbers_Equal()
+        {
+            double a = 121, b = 11;
+            double expected = 11;
+            double actual = Operations.Division(a, b);
+            Assert.AreEqual(expected, actual);
+        }
+
+        [Test]
+        public void Summ_Numbers_NotEqual()
```
Comment (IvanKrt) [inline NUnitTestProject1/OperationTest.cs]: Should be grouped by the method. Sum should be first

File: NUnitTestProject1/OperationTest.cs [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+            double actual = Operations.Division(a, b);
+            Assert.AreNotEqual(expected, actual);
+        }
+        [Test]
+        public void Division_DivisionByZero_Infinity()
+        {
+            double a = 1, b = 0;
+            double expected = Double.PositiveInfinity;
+            double actual = Operations.Division(a, b);
+            Assert.AreEqual(expected, actual);
```
Comment (IvanKrt) [inline NUnitTestProject1/OperationTest.cs]: What should you return to the user? Infinity or message that can`t be deleted by zero?

Comment (AlexeyScherbina) [reply NUnitTestProject1/OperationTest.cs]: I think, method should return Infinity. Message to user can be generated after the method returned value if it is required.

**Pull request merged**